src/layerstack.stanza

```diff
@@ -278,7 +302,7 @@ public defstruct LayerStack :
   description:Maybe<String>
 
   layers:Vector<LayerSpec> with: (
-    updater => sub-layers
+    setter => set-layers
```

Comment:
Why are we making this change ?

What is driving a need for mutability here ?

Reply:
To support the `set` function (since we have the `get` function):

```
; Test: test-stack[0] = new-stack
deftest(layerstack) test-set :
  #EXPECT(test-stack[0] == copper-3oz)
  test-stack[0] = copper-1oz
  #EXPECT(test-stack[0] == copper-1oz)

  #EXPECT(test-stack[1] == core-FR4-2)
  test-stack[1] = core-FR4-1
  #EXPECT(test-stack[1] == core-FR4-1)
```

Reply:
I would prefer we not implement this as a setter. The `sub-*` method allows this to be a transformation but maintains immutability. 

Additionally - we can very easily add this without breaking the ABI if we find that we need it later.